tests/rename_all_env.rs

```diff
@@ -0,0 +1,46 @@
+mod utils;
+
+use structopt::StructOpt;
+use utils::*;
+
+#[test]
+fn it_works() {
+    #[derive(Debug, PartialEq, StructOpt)]
+    #[structopt(rename_all_env = "kebab")]
+    struct BehaviorModel {
+        #[structopt(env)]
+        be_nice: String,
+    }
+
+    let help = get_help::<BehaviorModel>();
+    assert!(help.contains("[env: be-nice=]"));
+}
+
+#[test]
+fn default_is_screaming() {
+    #[derive(Debug, PartialEq, StructOpt)]
+    struct BehaviorModel {
+        #[structopt(env)]
+        be_nice: String,
+    }
+
+    let help = get_help::<BehaviorModel>();
+    assert!(help.contains("[env: BE_NICE=]"));
+}
+
+#[test]
+fn overridable() {
+    #[derive(Debug, PartialEq, StructOpt)]
+    #[structopt(rename_all_env = "kebab")]
+    struct BehaviorModel {
+        #[structopt(env)]
+        be_nice: String,
+
+        #[structopt(rename_all_env = "pascal", env)]
```

Comment:
I think in this case, it should be `rename_env`, because there is no "all" in this context.

Reply:
I don't quite understand. Do you propose rename the attr to `#[structopt(rename_env)]`?

Reply:
Yep, to match serde `rename` and `rename_all`. Now a local rename is, I think, not so interesting.

Reply:
>  to match serde `rename` and `rename_all`

You seem to be misunderstanding this. 

This attr is not meant to match `serde`'s `rename`/`rename_all`, it's meant to match `structopt`'s `rename_all`.

This specific piece of code tests the "override" feature for `rename_env_all` 
```rust
// we want all the derived flags here to be pascal case
#[structopt(rename_all = "pascal")]
// we also want the derived env fallbacks to use kebab case
#[structopt(rename_all_env = "kebab")]
enum Subcommands {
    // the stuff inside inherits parent renaming policy f
    // long, short, and name will be renamed to pascal case
    // env will be renamed to kebab case
    SetOne { /* ... */ },

    // We want this stuff to be different 
    // we *override* the setting, reapplying the same attributes with different values
    #[structopt(rename_all = "screaming snake", rename_all_env = "camel")]
    SetTwo { ... }

    // all the rest is not affected by the reapplying 
    // but it *is* affected by the parent settings
}
```

This is how you match `structopt`'s attrs to `serde`'s (roughly)
```
#[serde(rename)] == #[structopt(name/short/long/env)] 

#[serde(rename_all)] == #[structopt(rename_all/rename_all_env)]
```

Reply:
Yeh, you're right.